lib/services/p2p/index.js

```diff
@@ -109,13 +108,13 @@ P2P.prototype.sendTransaction = function(tx) {
 
 
 P2P.prototype.start = function(callback) {
-
   var self = this;
-  self._initCache();
-  self._initPool();
-  this._setListeners();
-  callback();
-
+  self._startBcoinIfNecessary(function(){
+    self._initCache();
+    self._initPool();
+    self._setListeners();
+    callback();
```

Comment:
startBcoinIfNec... 

I think the callback on 116 here may be unnecessary. Any function passed through will executed after Bcoin finishes syncing and 'full' is fired. But the cb does not take any parameters. I can see:

`() => { console.log('finished') }`;

working here. But what were your thoughts?

That said, this is benign and I am unable to tell what is going on in the scaffolding that could be passing a CB here or what its significance would be